modules/gatsby/src/components/layout/header.jsx

```diff
@@ -35,6 +36,9 @@ import {
   HeaderMenu,
   HeaderMenuLink
 } from '../styled/';
+import {navigate} from 'gatsby';
+import {Search} from 'baseui/icon';
+import {Input} from 'baseui/input';
 
 import GithubStars from '../github/github-stars.jsx';
```

Comment:
Unrelated... any chance we can get github stars back?

Reply:
oh, i have to remove those extra lines too. I was experimenting with having a search bar in the top header. 

we'll need to integrate that in the design. Design of the home page has a lot of room for improvement. The header is ok but home page is a mess.